### DISCUSSION TITLE
HELP HELP ! I sent WEth Solana back to Metamask Eth on Wormhole! Please help? Token not show up?

### DISCUSSION THREAD

**Discussion by Backsomeday22112010** [wormhole-foundation/wormhole#420] [Problems / bugs] [answered]
I sent wWETH (wormhole) from solana back to etherium. Transaction show success but did see token back to my metamask eth wallet 
Please help HELP

https://explorer.solana.com/tx/4fJXF9exQoXFrf7N7DBwCjXVYFVMEGrnh2DKRANGxhUf8KpirBM1Fe7rDeGPJx9Ho1kD5BJzfvHnwg9J9ouAVvH6

**Comment by leoluk** [comment, marked as answer]
The transaction was successfully processed by the Wormhole network, but the client-side transaction submission may have failed. Try clearing your cache and reloading wormholebridge.com to see if the transaction can be completed there. Make sure both wallets are connected, and select both tokens from the picker.

If wormholebridge.com doesn't work, you can manually submit the signed VAA bytes below to the submitVAA method on the Wormhole contract: https://etherscan.io/address/0xf92cd566ea4864356c5491c177a430c222d7e678#writeContract

---

Digest (for reference only): bf4791b89daa659fc7f7768b521360dd64fb4bda35b6efe4e01004953e4e4089

Signed VAA bytes to submit to the contract:

> 0x01000000010d006fdfb0cfa81020a409261295e4db49bd30ecdc3d94080c7ec0205537145d408c0d754fd21666e9769eb93b413fdbb311afc3daaff45672e276641b0c39a236d7010189c1538ae1a4e753f724590b2799f2f7c4ffc2ae9dfbb15acd6b39d51c5d61232f664fbb52190f798fa14f69406c558b7c31f4cb706202679462f57bf7a9c342010248ff7a36a65ecbcf576b243de906c2306bbe05065e7cdea6c14f585644fb41d601f95ad8f1388429ecef5e23b9c3a57eec8f2a290a37e1d8a40276d85e100ced0004398c974d5dfa6febe9658b199829cc555e9b8f68531ef51ea14660128bb1aa09133e637b0e6c431e29806b27c120534198b43f90a5ab2e10b67ded68593c061501051b254bfae45101308298a363b46c5f9d763492fce94f42bcee23b6836ea0f7534992abb8d412d8318b279b6a4692ad7d18620d630f82cb1b42a19ba8802b81250106fd06a8e922164ac16854a5b9847659d8ecae3e5aa631da814af8e0f3742e10fa14891d0508697154602d090d6c0d8ecd0337387d5e8b3d128446043b945bb25e010749364b855e8cd684fe1a521ec70459cf4416dd614a6bbbcb9ad3ba93487aab87633eaee31894ecd017e9a4a38a0d83b8c342b74baf3ddf1ca6ad06c6db014e270008bd0845b86af2fa735e345d57d41b24d534e93ae795fe7e7e9742f608d5b4cd4a0018a1030cbfb723a869b78fa4268dc717f7cc9ec3b4a09888d3d480543452aa000b82c010354ae242b2b39dcf8be3078091752af6cabcc9294fc8c91354d539f5fb10d1a37c8cb55f28c66a1540be9037502299ffc78d8564cc5e2a406e6117fffb000d0b1ac973e6270720ff881626e5ad1cf9ab8a91e09833560535cc1b2e25ea976833cda187022d8712cf8528725edc5fd2a72ce80c8913130d521258db44333a42010fa068c5b8dcdd02af806ad0bb2caa1a4d261771a4472f058f232bc8c62d6457c07c902ff39f2ac17372d05e4b29f2e5ca58f6814df7ea72ba8ce1a5823fe6ea010111973df28dd4f63b06fc79f32af918c538ddd8382cfbff882302bb52aac562377a4a138c90cb43baad1f57e6f436d250bbcd1eb06aaac1e1b5555bbca52b9f122500126feb7e7057333a5ed343916e0f9c9d7a87c2b83b51edea1e6b58670d668284d016638560ea90ddf291b4c2bedec3379b8ea59738f6a2a40063f86bcedc83dbe700613c63d2100000cdbe0102661d0ad5ef7906f29261f41997c50f7ee698a3a50ebaa75d08276433307737b500000000000000000000000091444b2b9fc973f1eccae3ad25d0fd30a1a4f83402000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc20900000000000000000000000000000000000000000000000000000001a13b8600

**Reply by Backsomeday22112010** [reply]
Thank you so much @leoluk i made it 
Can you please help me w VAA of the transaction below also

https://explorer.solana.com/tx/3QQWJqWpEZuWD13a5vjMEtBkv7SDWXYcRioyJpTLo5uqabfhvgofm1jvjJnS5nnEVmNVvNzEQQ83v815VL9V1btk


May i buy you something as my appreciation your help? Please send me your wallet address

**Comment by Backsomeday22112010** [comment]
Hi LeoThank you so much i make it Can you please help me w VAA of the transaction below also
https://explorer.solana.com/tx/3QQWJqWpEZuWD13a5vjMEtBkv7SDWXYcRioyJpTLo5uqabfhvgofm1jvjJnS5nnEVmNVvNzEQQ83v815VL9V1btk


May i buy you something as my appreaciation your help? Please send me your wallet address
Thanks again Leo
    On Monday, September 13, 2021, 03:44:21 AM PDT, Leopold Schabel ***@***.***> wrote:  
 
 


The transaction was successfully processed by the Wormhole network, but the client-side transaction submission may have failed. Try clearing your cache and reloading wormholebridge.com to see if the transaction can be completed there. Make sure both wallets are connected, and select both tokens from the picker.

If wormholebridge.com doesn't work, you can manually submit the signed VAA bytes below to the submitVAA method on the Wormhole contract: https://etherscan.io/address/0xf92cd566ea4864356c5491c177a430c222d7e678#writeContract

Digest (for reference only): bf4791b89daa659fc7f7768b521360dd64fb4bda35b6efe4e01004953e4e4089

Signed VAA bytes to submit to the contract:


0x01000000010d006fdfb0cfa81020a409261295e4db49bd30ecdc3d94080c7ec0205537145d408c0d754fd21666e9769eb93b413fdbb311afc3daaff45672e276641b0c39a236d7010189c1538ae1a4e753f724590b2799f2f7c4ffc2ae9dfbb15acd6b39d51c5d61232f664fbb52190f798fa14f69406c558b7c31f4cb706202679462f57bf7a9c342010248ff7a36a65ecbcf576b243de906c2306bbe05065e7cdea6c14f585644fb41d601f95ad8f1388429ecef5e23b9c3a57eec8f2a290a37e1d8a40276d85e100ced0004398c974d5dfa6febe9658b199829cc555e9b8f68531ef51ea14660128bb1aa09133e637b0e6c431e29806b27c120534198b43f90a5ab2e10b67ded68593c061501051b254bfae45101308298a363b46c5f9d763492fce94f42bcee23b6836ea0f7534992abb8d412d8318b279b6a4692ad7d18620d630f82cb1b42a19ba8802b81250106fd06a8e922164ac16854a5b9847659d8ecae3e5aa631da814af8e0f3742e10fa14891d0508697154602d090d6c0d8ecd0337387d5e8b3d128446043b945bb25e010749364b855e8cd684fe1a521ec70459cf4416dd614a6bbbcb9ad3ba93487aab87633eaee31894ecd017e9a4a38a0d83b8c342b74baf3ddf1ca6ad06c6db014e270008bd0845b86af2fa735e345d57d41b24d534e93ae795fe7e7e9742f608d5b4cd4a0018a1030cbfb723a869b78fa4268dc717f7cc9ec3b4a09888d3d480543452aa000b82c010354ae242b2b39dcf8be3078091752af6cabcc9294fc8c91354d539f5fb10d1a37c8cb55f28c66a1540be9037502299ffc78d8564cc5e2a406e6117fffb000d0b1ac973e6270720ff881626e5ad1cf9ab8a91e09833560535cc1b2e25ea976833cda187022d8712cf8528725edc5fd2a72ce80c8913130d521258db44333a42010fa068c5b8dcdd02af806ad0bb2caa1a4d261771a4472f058f232bc8c62d6457c07c902ff39f2ac17372d05e4b29f2e5ca58f6814df7ea72ba8ce1a5823fe6ea010111973df28dd4f63b06fc79f32af918c538ddd8382cfbff882302bb52aac562377a4a138c90cb43baad1f57e6f436d250bbcd1eb06aaac1e1b5555bbca52b9f122500126feb7e7057333a5ed343916e0f9c9d7a87c2b83b51edea1e6b58670d668284d016638560ea90ddf291b4c2bedec3379b8ea59738f6a2a40063f86bcedc83dbe700613c63d2100000cdbe0102661d0ad5ef7906f29261f41997c50f7ee698a3a50ebaa75d08276433307737b500000000000000000000000091444b2b9fc973f1eccae3ad25d0fd30a1a4f83402000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc20900000000000000000000000000000000000000000000000000000001a13b8600

**Reply by leoluk** [reply]
Sure, here's the VAA:

> 0x01000000010d0076d7c29f7adcda906e8a6de5422ab91dcc0bc82a05d012080df7a0e2647fcc90575902968aa87012e18a49e141f617ecc6eb52a160963e97b676257e0954596e0001930d06eb85779c05c0bd9a2fd140de1ae25eb62b3ff2564b2740ce2de25f3b4728f71156df71e85c72ca17ca41c42625c3a08b24c727d74c092a99cd36aac0c301022fb6f912bf1d6349e2b82e61538e46ffe7c55cf4339fcc28119da502d6dfb8134a17393953f97c6c27c7bcc993e69082d92ad65547d18fb75580db5b1d5f71380003650f3953159d71c917b3944e75458c88d05348807d7f751677b34a2d95bef02769a8050601d63af9dbf4c225cd093897daa2de125ddab4702a02c7dbc02406bd0004b7d0291bf6e27ecd65ceb893cf7494e0b6ad1f8378667bceec4fa2bae65012ab4372588dc542ade7bf53f83af510bb24f58e79b620bf3331b507a8a6324914fc0005a5f65ced5742f6139b15120ab52627b028bbf8265942e098b7b042a8f0c103050673cd3e74f695b1bd5ac7b2f904f27d5b9618ea400f77702589415dc242f994000640fa5dad0609100b7a09637c8528c3a1c2500d81a3ad871121c3f86c10acf1143a624855266e4b70647e9167101b89168c2e330f88d2541c6bdba311602787d10107e89994f0e2e409a3021b4a7a99b8dd24c8f5b3a468109ac07cbec93a312913c458dc58a37ff010033ca6d373206795b6dce148785cf7f6f7b151f5220c9777170008211de8f7d25bd4b95d7f560f2cd400fb178d8c625b5ee8312c4e3227ef726ee635ec743802d97e905e78b6472d376681af8da1034db7b483051be5c48dbd94e1010b48619a0bbf3fd688157cd739de18bfae126ae1267bba2fdc1d4e7c16a560fa1e75b709b9d33988c8c71197eda5d65a936700cf27bcdc99b83066a37fb36c7727010d7037a54ad8d477e137004ab975264c504acd2c5bd9684c61bdb3922cdd1ebd2d3594d2248df7faacf1697a27de18c4461cf2d6fabc9a6c78dbb6437e3a6440d50111893bdf19925a9edc994af2507201f49e733063bafff6192aadf7bfceae275c460f8a1ead665aeac3a2a3ddf464ef4ad6f77259798a0d6839d46f37bb4bba92520012fe1536341e576c302e6868ffaf7fefe15706b675af131c146c727f2b64e813e5554249f8bf403eb48b65d02fd9bca370a25d0d522b7bda83d2ca571b7296100e00613c64b610000159980102661d0ad5ef7906f29261f41997c50f7ee698a3a50ebaa75d08276433307737b500000000000000000000000091444b2b9fc973f1eccae3ad25d0fd30a1a4f83402000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc209000000000000000000000000000000000000000000000000000000003b9aca00
